src/Global/molecules/TopBar.tsx

```diff
@@ -71,159 +74,63 @@ const DrawerToggle = styled.button.attrs({ type: 'button' })<{ isActive: boolean
   outline: none;
   cursor: pointer;
 
-  ${({theme}) => css`
+  ${({ theme }) => css`
     border-bottom: 5px solid ${theme.colors.menu.indicator};
 
     &:hover {
       border-bottom-color: ${theme.colors.menu.hover};
     }
 
     transition: border ${theme.animation.speed.normal} ${theme.animation.easing.primary.easeInOut};
-
   `};
 
-  ${({isActive, theme}) => isActive && css`
+  ${({ isActive, theme }) => isActive && css`
     &, &:hover {
       border-bottom-color: ${theme.colors.menu.active};
     }
   `}
 `;
 
-const Drawer = styled.div<{ isOpen : boolean }>`
-
-  font-family: ${({theme}) => theme.fontFamily.ui};
+const Drawer = styled.div<{ isOpen: boolean }>`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
 
   position: fixed;
   right: -10px;
   top: 65px;
   bottom: 0;
-  background: ${({theme}) => theme.styles.global.mainMenu.background};
-  border-left: ${({theme: {colors}}) => colors.divider} 1px solid;
-  width: 200px;
+  background: ${({ theme }) => theme.styles.global.mainMenu.background};
+  border-left: ${({ theme: { colors } }) => colors.divider} 1px solid;
+  width: 250px;
```

Comment:
The design in zepling is 300px for this part in desktop but it looks very empty for the user options drawer.
However it the alerts designs doesn't fit correctly in 200px.. decided to go 250px but please advice

Reply:
ok, sounds better. Will update :)